### PR TITLE
Support ms:mexecute

### PR DESCRIPTION
**Pull request by Leodau**
## What does this PR do?
Implements support for the newly added **ms:mexecute**

### How should this be manually tested?

```
const actions = [
    { 'action': 'set', 'args': { '_id': 'list:a', 'body': { 'value': 1, 'ex': 100, 'nx': true } } },
    { 'action': 'get', 'args': { '_id': 'list:a' } },
    { 'action': 'del', 'args': { 'body': { 'keys': ['list:a'] } } }];

  // Should Print : "[ [ null, 'OK' ], [ null, '1' ], [ null, 1 ] ]"
  console.log(await kuzzle.ms.mexecute(actions));
```

### Boyscout

- Applied _one-var_ esrule in the memoryStorage file.

### CODE REVIEW

File: doc/7/controllers/ms/mexecute/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+code: true
+type: page
+title: mexecute
+---
+
+# mexecute
+
+Allows the execution of multiple commands or 'actions' in a single step.
+
+It creates a redis **transaction** block and **executes** it immediately, all actions will be executed sequentially and as a single atomic and isolated operation.
```
Comment (Aschen) [inline doc/7/controllers/ms/mexecute/index.md:11]:
```suggestion
It creates a Redis **transaction** block and **executes** it immediately, all actions will be executed sequentially and as a single atomic and isolated operation.
```

File: doc/7/controllers/ms/mexecute/snippets/mexecute.test.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: ms#mexecute
+description: Executes multiple commands in a single step
+hooks:
+  before: curl -X POST kuzzle:7512/ms/_flushdb
+  after:
+template: default
+expected: "\\[ \\[ null, 'OK' ], \\[ null, '1' ], \\[ null, 1 ] ]"
+sdk: js
+version: 7
```
Comment (Aschen) [inline doc/7/controllers/ms/mexecute/snippets/mexecute.test.yml:9]:
```suggestion
```

File: doc/7/controllers/ms/mexecute/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+[[_Redis documentation_]](https://redis.io/topics/transactions)
+
+::: warning
+Only already supported actions can be executed using **mexecute**.
```
[the remark this answers is not in the document]
Comment (Aschen) [reply doc/7/controllers/ms/mexecute/index.md:16]: It's the memory storage controller actions

[the remark this answers is not in the document]
Comment (Leodau) [reply doc/7/controllers/ms/mexecute/index.md:16]:
```suggestion
Only already valid actions of the memoryStorage controller can be executed using **mexecute**.
```

**Pull request merged**